test/testvm.py

```diff
@@ -98,6 +125,19 @@ def wait_ssh(self):
             time.sleep(1)
         return False
 
+    def wait_user_login(self):
+        """Wait until logging in as non-root works"""
+        tries_left = 5
+        while (tries_left > 0):
+            try:
+                self.execute("! test -f /run/nologin")
+                return
+            except:
+                pass
+            tries_left = tries_left - 1
+            time.sleep(1)
+        raise Failure("Timed out waiting for /run/nologin to disappear")
```

Comment:
We do both.  Ssh starts working for root before it starts working for non-root.  See 1bb4ba35.